packages/core/src/crypto/jose/jwk/ecCompression.ts

```diff
@@ -2,23 +2,26 @@
  * Based on https://github.com/transmute-industries/verifiable-data/blob/main/packages/web-crypto-key-pair/src/compression/ec-compression.ts
  */
 
+// TODO(crypto): can remove this?
```

Comment:
Yeah i think so. Would be great as part of 0.6. Can you remove and add a breaking changeset for this?